tests/sentry/api/endpoints/test_project_stacktrace_link.py

```diff
@@ -162,13 +162,13 @@ def test_file_not_found_and_munge_frame_fallback_not_found(self, mock_integratio
             },
         )
 
-        assert response.data["config"] == self.expected_configurations(self.code_mapping2)
+        assert response.data["config"] == self.expected_configurations(self.code_mapping1)
```

Comment:
This is now correctly returning the configuration which matches rather than the last one.

This is because we were iterating through _every_ code mapping rather than the ones that match the stack root.